### DISCUSSION TITLE
Cannot connect to iOS device after updating to iOS 15.4.1

### DISCUSSION THREAD

**Discussion by todor-rusev** [airsdk/Adobe-Runtime-Support#1919] [Q&A] [answered]
Hi,
I am not able to run my AIR project on iOS device, after updating to iOS 15.4.1.
I am using IntelliJ and an error popup occurres every time with the following message: "_No iOS devices connected to the computer_". The compilation of the project and the package complete with no error.

The following command: **adt -devices -platform iOS**
gives me: 
**_ERROR: Could not connect to lockdownd: Invalid HostID (-21)
Could not read device info**

Note: the device is not jailbroken and this issue has not occured before the update of the device. 
The package was successfully installed on another device with an older version.

I am using:
Windows 10
AIRSDK for Windows - **33.1.1.856**
**iPhone 8** - version **15.4.1**
IntelliJ IDEA 2022.1.1 (Ultimate Edition)

Thank you in advance!!

**Comment by ajwfrost** [comment, marked as answer]
Can you confirm that iTunes is able to connect to it?

The other thing to check is to go to the AIR SDK folder "lib\aot\bin\iOSBin" in a command prompt, and run "ideviceinfo.exe" to see if that can detect the phone.

thanks

**Comment by todor-rusev** [comment]
Hello, 

I have just tested to connect the iPhone with ITunes and it worked. 
Then I tried to install the app using IntelliJ IDE and everything is back to normal. 

Many thanks for your assistance!! :)
